### PR TITLE
feat: onboard large scale notification bot sample

### PR DESCRIPTION
**Pull request by yiqing-zhao**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/24539641

### CODE REVIEW

File: large-scale-notification/assets/sampleDemo.png
Comment (tecton) [inline large-scale-notification/assets/sampleDemo.png:1]: You could rename it to `thumbnail.png`.

File: large-scale-notification/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com"
+  },
+  "dependencies": {
+    "@azure/data-tables": "^13.2.2",
+    "@azure/service-bus": "^7.9.0",
+    "@microsoft/adaptivecards-tools": "^1.0.0",
+    "@microsoft/teamsfx": "2.2.3-beta.2023081506.0",
```
Comment (tecton) [inline large-scale-notification/package.json:29]: Do we have stable release of SDK?

File: large-scale-notification/script/createTableForLocalDebug.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import {
+  TableServiceClient,
+  AzureNamedKeyCredential,
+} from "@azure/data-tables";
+
+const account = "devstoreaccount1";
+const accountKey =
```
Comment (tecton) [inline large-scale-notification/script/createTableForLocalDebug.ts:7]: Is this a known key? You'd better add a comment or remove it if not.

File: large-scale-notification/assets/sampleDemo.png
Comment (yiqing-zhao) [reply large-scale-notification/assets/sampleDemo.png:1]: Updated, thanks!

File: large-scale-notification/package.json (hunk 1 above)
Comment (yiqing-zhao) [reply large-scale-notification/package.json]: Stable SDK was released just now, updated to use stable SDK.

File: large-scale-notification/script/createTableForLocalDebug.ts (hunk 2 above)
Comment (yiqing-zhao) [reply large-scale-notification/script/createTableForLocalDebug.ts:8]: This is a well-known key listed [here](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio#well-known-storage-account-and-key). Added a comment in the file, thanks!

**Review by tecton — APPROVED** [review]
Please wait for testing.

**Review by tecton — APPROVED** [review]

**Pull request merged**